Extensions/TileMap/tilemapruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * The PIXI.js renderer for the Tile map runtime object.
+ *
+ * @class TileMapRuntimeObjectPixiRenderer
+ * @constructor
+ * @param {gdjs.TileMapRuntimeObject} runtimeObject The object to render
+ * @param {gdjs.RuntimeScene} runtimeScene The gdjs.RuntimeScene in which the object is
+ */
+gdjs.TileMapRuntimeObjectPixiRenderer = function(runtimeObject, runtimeScene) {
+  this._object = runtimeObject;
+  this._runtimeScene = runtimeScene;
+
+  // Load (or reset)
+  if (this._pixiObject === undefined) {
+    this._pixiObject = new PIXI.tilemap.CompositeRectTileLayer(0);
+    this._pixiObject.tileAnim = [0, 0];
+  } else {
+    // Run updates a single time once loaded here
```

Comment:
Not sure I understand this comment?

Reply:
accidentally copied it from another extension. Will remove this